README.md

```diff
@@ -1,16 +1,305 @@
+	}
+
+	areaMessage() {
+		return `The area of this square is ${this.area()}`;
+	}
+}
+square.area(); // => 25
+square.areaMessage(); // => LOG: The area of this square is 25
+```
+
+In the `class` above, we can access `area()` directly, or use it to provide
+dynamic content for other methods. These methods are the most common - they act
+as the 'behaviors' of a `class` instance.
+
+## Static Methods
+
+Static methods are `class` level methods - they are not callable on instances of
+a `class`, only the `class` itself. These are often used in 'utility' `class`es -
+`class`es that encapsulate a set of related methods but don't need to be
+made into instances. For example, we could write a `CommonMath` `class` that
+stores a series of math related methods:
+
+```js
+class CommonMath {
+	static triple(number) {
+		return number * number * number;
+	}
+
+	static findHypotenuse(a, b) {
+		return Math.sqrt(a * a + b * b);
+	}
+}
+```
+
+To access, these static methods:
+
+```js
+let num = CommonMath.triple(3);
+num; // => 27
+let c = CommonMath.findHypotenuse(3, 4);
+c; // => 5
+```
+
+This sort of `class` might be useful in many different situations, but we don't
+ever need an _instance_ of it.
+
+## Define `get` Keyword in JavaScript Class Context
+
+Often, when writing methods for a `class`, we want to return information
+derived from that instance's properties. In the Square `class` example, `area()`
+returns a calculation based on `this.sideLength`, and `areaMessage()` returns
+a `String`.
+
+In modern JavaScript, new syntax, `get`, has been introduced to `class`es
+to distinguish methods which serve a specific purpose to 'get' data from an
+instance.
+
+The `get` keyword turns a method into a 'pseudo-property', that is - it allows
+us to write a method that interacts like a property. To use `get`, write a
+`class` method like normal, preceded by `get`:
+
+```js
+class Square {
+	constructor(sideLength) {
+		this.sideLength = sideLength;
+	}
+
+	get area() {
+		return this.sideLength * this.sideLength;
+	}
+}
+```
+
+As a result of this, `area` will now be available as though it is a
+property just like `sideLength`:
+
+```js
+let square = new Square(5);
+square.sideLength; // => 5
+square.area; // => 25
+```
+
+If you try to use `this.area()`, you'll receive a TypeError - `area` is no
+longer considered a function!
+
+This may seem strange - you could also just write the following and achieve the
+same result:
+
+```js
+class Square {
+	constructor(sideLength) {
+		this.sideLength = sideLength;
+		this.area = sideLength * sideLength;
+	}
+}
+```
+
+This is valid code, but what we've done is load our `constructor` with more
+calculations. The main benefit to using `get` is that your `area` calculation
+isn't actually run until it is accessed. The 'cost' of calculating is offset,
+and may not be needed at all if `this.area` is never called. For calculations or
+processes that are costly, this can be a handy tool.
```

Comment:
Something about this I feel should be emphasized in some way. This is a pretty important point.